pkgs/lib/xtb-python/default.nix

```diff
@@ -33,13 +33,14 @@ buildPythonPackage rec {
 
   # Build a C module to interface XTB.
   preBuild = ''
-    meson setup build --prefix=$PWD --default-library=shared
+    meson setup build --prefix=$(pwd) --default-library=shared
     ninja -C build install
+    cp ./${python.passthru.sitePackages}/_libxtb.*.so ./xtb/.
```

Comment:
Something is going wrong here:
```
cp: missing destination file operand after './xtb/.'
```

Reply:
I guess I have an idea. What version does meson report during build? 1.57 or 1.60? This copy works around a changed behaviour of meson with the update in nixpkgs, which changes the installation directory. The original version was OK with 1.57, the version in this branch is OK with 1.60. We can add a conditional, to allow both, but this version fixes #151 at the cost of breaking the current pin.

Reply:
According to Hydra it is `meson-0.57.1` (from the internally pinned nixpkgs). Should we just stick to the pinned version?

Reply:
Hm, good question. I would be in favour of updating the pin to have meson 1.60, so that we don't run into this problem later again.
Alternatively, I could replace it with a `find` command, which should work for both meson versions?

Reply:
Sorry, I got mixed up here. Yes, the purpose is to upgrade the pinning. This version working with meson 1.6 is okay.

Reply:
(these version bugs are nasty)